app/models/translation.rb

```diff
@@ -4,4 +4,48 @@ class Translation < ActiveRecord::Base
 	attr_accessible :user, :video
 
 	validates_presence_of :video
+
+	def video
+		Video.find(self.video_id)
```

Comment:
Also, you don't need this method. It has an association, so `translation.video` will magically return its video.